### DISCUSSION TITLE
Hide disabled table row actions when all rows are disabled

### DISCUSSION THREAD

**Discussion by dmason30** [laravel/nova-issues#5002] [Ideas & Feature Requests]
- Nova Version: 4.18.0
- Patch: https://gist.github.com/dmason30/421c0a36b01789447a6f8819db5a5694

### Description:
I appreciate that this change came from the discussion I started, and I am grateful a solution has been implemented: https://github.com/laravel/nova-issues/discussions/4844

However, I didn't expect it to show all buttons all the time even if that resource will never have a row where that action is enabled. For example, this resource can **never** be `viewed` or `edited` but can conditionally be `deleted`:

<img width="157" alt="image" src="https://user-images.githubusercontent.com/20278756/199370937-1310922a-ca13-4b15-ab62-8d8ee2d7aab5.png">

With the patch (proof of concept) linked above, it will hide a column of icons if all rows are disabled for each specific action. For example, this resource instead would display like this as only `delete` has some rows where the action is enabled:
<img width="165" alt="image" src="https://user-images.githubusercontent.com/20278756/199371289-06634f9b-1886-4f59-b1a0-fdf48d20fec1.png">

**Comment by kusab85** [comment]
Comment for discussion bump.

**Comment by crynobone** [comment]
Implemented in Nova 4.19.0
